guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -6,14 +6,23 @@ Configure a downstream {ProjectServer} to synchronize repositories from a connec
 .Prerequisites
 * A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.
 * You imported the subscriptions manifest on both the upstream and downstream {ProjectServer}.
-* You enabled the required repositories for the organization on the upstream {ProjectServer}.
-* You have user permissions to create product repositories and organizations on the downstream {ProjectServer}.
+* On the upstream {ProjectServer}, you enabled the required repositories for the organization.
+* The upstream user is an admin or has the following permissions:
+** `view_organizations`
+** `view_products`
+** `edit_organizations` (to download the CA certificate)
+** `view_lifecycle_environments`
+** `view_content_views`
+* On the downstream {ProjectServer}, you have imported the SSL certificate of the upstream {ProjectServer} using the contents of `http://_upstream-{foreman-example-com}_/pub/katello-server-ca.crt`.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing SSL Certificates] in the _Content Management Guide_.
+* The downstream user is an admin or has the permissions to create product repositories and organizations.
```

Comment:
@rmynar Can I get an ack on this, pls? :)

Reply:
This looks OK. ACK from me :)